.editorconfig

```diff
@@ -10,3 +10,4 @@ trim_trailing_whitespace = true
 
 [*.md]
 indent_size = 2
+trim_trailing_whitespace = false
```

Comment:
Please think twice about how to do a hard line break in Markdown before you write this rule...

Reply:
An existing file already uses this format

```
Line<space><space>
Line
```

Reply:
Oh, my eye is broken... I see false as true. :(